cart/cart.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./cart.css">
+    <title>Cart | Vivir fashion</title>
+</head>
+<body>
+    <div class="container product-navbar">
+        <div class="navbar">
+            <div class="logo">
+             <a href="../home/index.html">vivir</a>
+            </div>
+            <nav>
+                <ul class="nav-pages">
+                    <li><a href="../index.html">Home</a></li>
+                    <li><a href="../products.html">Products</a></li>
+                    <li><a style="color: var(--orange);
```

Comment:
avoid using inline styling, for good practices

Reply:
Thanks for suggesting, made the changes, please approve this PR